src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+            // then there's nothing left to visit anyway.
+            if (receiverOpt == null)
```

Comment:
> Do you know of any other calls that pass null?

We can certainly check other call sites. However, given that implementation of our visitors usually follow a pattern when it is Ok to visit "null". Look, for example, how ```Visit(BoundNode node)``` is implemented. Basically, visiting null is a no-op, not a crash. Given this, it is very easy to fall into a trap when, for some reason, a code is switched from calling regular Visit to calling VisitRValue/VisitLvalue.

>  I'm not sure what a hardened implementation would look like.

I would use VisitAlways(BoundNode node) as an example. If the input is null, give it back and get out.